Build failure on commit 12a5922

When building this project (as of commit 12a5922), I get the following error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project JSONata4Java: Compilation failure
[ERROR] /tmp/buildss/JSONata4Java/src/main/java/com/api/jsonata4java/expressions/functions/EvalFunction.java:[87,27] exception com.api.jsonata4java.expressions.EvaluateException is never thrown in body of corresponding try statement
```

--- a/src/main/java/com/api/jsonata4java/expressions/functions/EvalFunction.java
+++ b/src/main/java/com/api/jsonata4java/expressions/functions/EvalFunction.java
@@ -23,7 +23,6 @@
 package com.api.jsonata4java.expressions.functions;
 
 import com.api.jsonata4java.Expression;
-import com.api.jsonata4java.expressions.EvaluateException;
 import com.api.jsonata4java.expressions.EvaluateRuntimeException;
 import com.api.jsonata4java.expressions.ExpressionsVisitor;
 import com.api.jsonata4java.expressions.ParseException;
@@ -84,7 +83,7 @@ public class EvalFunction extends FunctionBase implements Function {
 			try {
 				Expression expr = Expression.jsonata(expression);
 				result = expr.evaluate(context);
-			} catch (EvaluateException | ParseException | IOException e) {
+			} catch (ParseException | IOException e) {
 				throw new EvaluateRuntimeException(ERR_ARG1BADTYPE);
 			}
 		} else {

